src/unc_tools.cpp

```diff
@@ -160,7 +160,7 @@ void dump_out(unsigned int type)
          fprintf(D_file, "  orig_col %zu\n", pc->orig_col);
          fprintf(D_file, "  orig_col_end %zu\n", pc->orig_col_end);
          fprintf(D_file, (pc->orig_prev_sp != 0) ? "  orig_prev_sp %u\n" : "", pc->orig_prev_sp);
-         fprintf(D_file, (pc->flags != 0) ? "  flags %zu\n" : "", pc->flags);
+         fprintf(D_file, (pc->flags != 0) ? "  flags %llu\n" : "", pc->flags);
```

Comment:
The `llu` is for unsigned `long long ints`, I do not where this could cause a problem but its not guaranteed that it will be equally sized as `uint64_t`. I would recommend the use of the more fitting C99 `PRIu64`. 
The problem with `PRIu64` on the other hand would be that Microsoft added support for that only with [VS2013}(https://blogs.msdn.microsoft.com/vcblog/2013/07/19/c99-library-support-in-visual-studio-2013/). 

@gmaurel what minimum VS version do we support?